tests/testthat/test-carehomes-check.R

```diff
@@ -48,6 +50,56 @@ test_that("everyone is infected when beta is large", {
 })
 
 
+test_that("noone stays in R, R_neg or PCR_neg if waning rate is very large", {
+  p <- carehomes_parameters(0, "england", beta_value = 0,
+                            waning_rate = 1e9)
```

Comment:
Could set `waning_rate = Inf` here, as typically all tests for rates (gammas) are at 0 or Inf. The test where beta is very large doesn't use a value of Inf because in that specific case as it causes NA values in compartments.